src/applications/caregivers/definitions/UIDefinitions/veteranUI.js

```diff
@@ -133,15 +135,14 @@ export const LastTreatmentFacilityAPIUI = {
 };
 
 export const PreferredFacilityAPIUI = {
-  'ui:title': 'VA health care services',
-  'ui:description': PreferredFacilityDescription,
-  veteranFacilityState: {
-    'ui:title': 'State',
-    'ui:required': () => true,
-  },
+  'ui:title': 'The Veteran’s VA health care facility',
+  'ui:description': PreferredFacilityAPIDescription,
   plannedClinic: {
-    'ui:title': 'VA medical center',
-    'ui:widget': VaMedicalCenter,
+    'ui:title': 'Search by city, state or postal code',
+    'ui:errorMessages': {
```

Comment:
I'm pretty sure we need to handle validation manually in the widget, so this may end up being rendered useless. Just make a note to remove if you find you have to manually validate.